pyfolio/plotting.py

```diff
@@ -30,18 +30,32 @@
 from . import pos
 
 
-def set_plot_defaults():
-    """
-    Sets default plotting/charting colors/styles.
-    """
+def plotting_context(func):
+    """Decorator to set plotting context during function call."""
+    def call_w_context(*args, **kwargs):
+        set_context = kwargs.pop('set_context', True)
+        if set_context:
+            with context():
+                return func(*args, **kwargs)
+        else:
+            return func(*args, **kwargs)
+    return call_w_context
+
+
+def context(context='notebook', font_scale=1.5, rc=None):
```

Comment:
Actually it is useful already, a user could do:
```python
with pf.plotting.context(font_scale=2):
    pf.create_tear_sheet(set_style=False)
```

I think that's a good way to use it. Will add docs.